examples/random/README.md

```diff
@@ -0,0 +1,43 @@
+# Random number generator sample
+
+This example demonstrates how to use scalar and vector random number generation.
+
+The computations are performed using Intel® oneAPI DPC++ library (oneDPL).
+
+| Optimized for                   | Description                                                                        |
+|---------------------------------|------------------------------------------------------------------------------------|
+| OS                              | Linux Ubuntu 18.04                                                                 |
+| Hardware                        | Skylake with GEN9 or newer                                                         |
+| Software                        | Intel® oneAPI DPC++ Compiler (beta)                                                |
```

Comment:
I think that we should align OS/Hardware/Software assumptions across all examples. 
But we can do that later probably

Reply:
I agree about Hardware and OS, but some examples require TBB and don't require DPC++ Compiler, others vice-versa because some example are host only (using TBB as a backend).